src/components/TopBar/Breadcrumb.js

```diff
@@ -0,0 +1,104 @@
+// @flow
+
+import React, { PureComponent } from 'react'
+import styled from 'styled-components'
+import type { Account } from '@ledgerhq/live-common/lib/types/account'
+import type { T } from 'types/common'
+import { compose } from 'redux'
+import { connect } from 'react-redux'
+import { createStructuredSelector } from 'reselect'
+import { push } from 'react-router-redux'
+import { translate } from 'react-i18next'
+
+import { accountsSelector } from '../../reducers/accounts'
+import Crumb from './Crumb'
+import Button from '../base/Button'
+
+type Props = {
+  pathname: string,
+  accounts: Account[],
+  push: string => void,
+  t: T,
+}
+
+const mapStateToProps = createStructuredSelector({
+  accounts: accountsSelector,
+})
+
+const mapDispatchToProps = {
+  push,
+}
+
+const Wrapper = styled.div`
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  > * {
+    font-family: 'Open Sans';
+    font-weight: 600;
+    font-size: 12px;
+    color: ${p => p.theme.colors.grey};
+  }
+
+  > :first-child {
+    padding: 0px;
+    &:hover {
+      background: transparent;
+      text-decoration: underline;
+    }
+  }
+`
+
+const Separator = styled.div`
+  &::after {
+    content: '/';
+    font-size: 13px;
+    color: ${p => p.theme.colors.fog};
+    padding: 0 15px;
+  }
+`
+
+class Breadcrumb extends PureComponent<Props> {
+  processItemsForDropdown = (items: any[]) =>
+    items.map(item => ({ key: item.id, label: item.id, account: item }))
+
+  render() {
+    const { t, push, pathname, accounts } = this.props
+    let tokenItems
+    let token
+
+    if (!pathname.startsWith('/account/')) return null
```

Comment:
I will suggest a rework of architecture of this component. we should use React Router.

suggest to have this move to:

```
components/TopBar/Breadcumb/index.js
```

and only doing the minimal routing needed.

look what's done here:

https://github.com/LedgerHQ/ledger-live-desktop/blob/topic/erc20/src/components/layout/Default.js#L120-L129

same logic can be done with this:

you can have a top level:

```js
                  <Switch>
                    <Route path="/account/:parentId/:id" component={AccountWithTokenBroadcrumb} />
                    <Route path="/account/:id" component={AccountBroadcrumb} />
                  </Switch>
```

or share the same component is possible too

```js

                  <Switch>
                    <Route path="/account/:parentId/:id" component={AccountBroadcrumb} />
                    <Route path="/account/:id" component={AccountBroadcrumb} />
                  </Switch>
```

but at least that would avoid all the ifs.

only that `AccountBroadcrumb` would then get Connected. 

could be done in this file:

```
components/TopBar/Breadcumb/Account.js
```

and you can actually take the same connect logic of https://github.com/LedgerHQ/ledger-live-desktop/blob/topic/erc20/src/components/AccountPage/index.js#L33 (that could be interesting to split into a dedicated selector that is basically this:

```js
const accountAndParentAccountSelector = (state, {
    match: {
      params: { id, parentId },
    },
  },
) => {
  const parentAccount: ?Account = parentId && accountSelector(state, { accountId: parentId })
  let account: ?(TokenAccount | Account)
  if (parentAccount) {
    const { tokenAccounts } = parentAccount
    if (tokenAccounts) {
      account = tokenAccounts.find(t => t.id === id)
    }
  } else {
    account = accountSelector(state, { accountId: id })
  }
}
```